File: updater/src/main/java/identifiers/WorldMapAreaData.java
package identifiers;

import utility.AbstractIdentifier;
import utility.ClassWrapper;
import utility.DependsOn;

@DependsOn(WorldMapArea.class)
public class WorldMapAreaData extends AbstractIdentifier {
    @Override
    public boolean isMatch(ClassWrapper classNode) {
        return classNode.isSuperClassEquals("WorldMapArea");
    }
}

File: find-names/src/main/java/mapper/identifiers/classes/WorldMapAreaData.kt
package mapper.identifiers.classes

import mapper.abstractclasses.IdentityMapper
import mapper.annotations.DependsOn
import mapper.predicateutilities.predicateOf
import mapper.wrappers.ClassWrapper

@DependsOn(WorldMapArea::class)
class WorldMapAreaData : IdentityMapper.Class() {

    override val predicate = predicateOf<ClassWrapper> { it.superType == type<WorldMapArea>() }
}